### ISSUE TITLE
VBA QA: Heading structure is not helpful to user

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#18222]
## Description
This is somewhat related to the ticket regarding "Appointments" showing with no content but ultimately because we have so many headings in the service locations, and it's all connected to other templates, they aren't hierarchical like they should be.

For example, the phone number under appointments should be an h6 but there is really no reason to be going that far down, which is why I'm advocating to just get rid of these headers here - they aren't necessary. 

The bigger issue is that having the heading level the same is potentially going to create more confusion because it's muddying the relationship of these elements

## Acceptance Criteria
- [ ] Determine if it is possible/feasible/desirable to update headings without touching VAMCs
- [ ] If so, determine how to update the headings for the best UX and a11y experience
- [ ] Requires design review
- [ ] Requires accessibility review

**Comment by davidmpickett** [comment]
This might need to be split into a few tickets to cover the various places where we are using headings that we want to move toward bold/strong text instead.

- VBA main page template 
  - Should be pretty small scope and could be launch blocking
  - I think Location and Contact information is the main area to fix
    - Address, phone numbers, and office hours 
- VBA & VAMC service accordions
  - I think we should fix these on VAMCs and VBA at the same time since they share partial templates
  - Possible fast follow?
- Other places on VAMCs
  - Etc.

**Comment by Agile6MSkinner** [comment]
To Dave's comment, let's split out another ticket for the Man Page Template, which will will consider launch blocking. This ticket will stay in the backlog for fast follow/national rollout

**Comment by laflannery** [comment]
To be clear, this ticket is referring to the heading structure within the accordions, as described in the description above. The headings outside the accordions, while eventually they could probably be improved, they aren't hindered the user's ability to understand the content.

**Comment by Agile6MSkinner** [comment]
After talking with @laflannery we are moving this to the national rollout epic because it is related to a more global issue and will require a lot of design discovery. It's not a high-priority issue, it just looks bad.

@thejordanwood Do you have an epic that this could be included in or a design discovery ticket that we could set as a pre-req?

**Comment by davidmpickett** [comment]
> @thejordanwood Do you have an epic that this could be included in or a design discovery ticket that we could set as a pre-req?

- @thejordanwood did some work on this before as part of #17342
- That ticket is part of a catchall epic for anything about service locations or service accordions #17350
- Likely would need a "Finalize design recommendations for Service Accordion visual hierarchy" ticket ala #17489

**Comment by davidmpickett** [comment]
> To be clear, this ticket is referring to the heading structure within the accordions, as described in the description above. The headings outside the accordions, while eventually they could probably be improved, they aren't hindering the user's ability to understand the content.

Updated ticket title to reflect this scope

**Comment by Agile6MSkinner** [comment]
@davidmpickett @thejordanwood Do we think that the work described here is discrete or will it be taken care of in other issues/tickets/epics?

**Comment by JonathanKatora** [comment]
@davidmpickett Is the original ask of this ticket still true? Laura's write up suggests that Headings under service accordians are <h6>. However I'm struggling to find headers much below <h4>

<img width="1233" height="551" alt="Image" src="https://github.com/user-attachments/assets/fbf4838a-2b92-4a29-8328-249ffee8f24c" />

**Comment by davidmpickett** [comment]
Yes. Right now the Appointments header and the Appointments phone number are both h4s, which implies they are separate pieces of information that are unrelated. But actually that phone number is related to Appointments specifically, vs other phone numbers in the accordion that aren't appointment related. So if we're using headings, we need more hierarchy, or we could remove a lot of the headings and just have things be bold

<img width="1233" height="551" alt="Image" src="https://github.com/user-attachments/assets/a6608b79-c139-4ba8-9ffc-54ee84d3c969" />

**Comment by JonathanKatora** [comment]
Got it. Thanks for the extra context.
